deps/patches/libgit2-mbedtls-verify.patch

```diff
@@ -1,99 +1,105 @@
-diff --git a/src/mbedtls_stream.c b/src/mbedtls_stream.c
-index bcc2e8b..4706102 100644
---- a/src/mbedtls_stream.c
-+++ b/src/mbedtls_stream.c
-@@ -221,82 +221,34 @@ static int ssl_teardown(mbedtls_ssl_context *ssl)
-     return ret;
+commit 1732ba4c3f6e62017c94e4c60dade9802da1a317
+Author: Curtis Vogt <curtis.vogt@gmail.com>
+Date:   Fri Jun 23 18:53:10 2017 -0500
+
+    Use mbedtls certificate verification
+
+diff --git a/src/streams/mbedtls.c b/src/streams/mbedtls.c
+index 0376ee4..e52afc8 100644
+--- a/src/streams/mbedtls.c
++++ b/src/streams/mbedtls.c
+@@ -228,82 +228,34 @@ static int ssl_teardown(mbedtls_ssl_context *ssl)
+ 	return ret;
  }
  
 -static int check_host_name(const char *name, const char *host)
 -{
--    if (!strcasecmp(name, host))
--        return 0;
+-	if (!strcasecmp(name, host))
```

Comment:
Looks like GitHub diff can't handle diffing diffs. The original patch removed the `check_host_name` function and used the mbedtls function for doing cert verification instead of rolling our own in libgit2